Modules/Core/Common/src/itkAnatomicalOrientation.cxx

```diff
@@ -83,9 +83,9 @@ AnatomicalOrientation::CreateFromPositiveStringEncoding(std::string str)
   auto                                                                        iter = stringToCode.find(str);
   if (iter == stringToCode.end())
   {
-    return AnatomicalOrientation(PositiveEnum::INVALID);
+    return { PositiveEnum::INVALID };
```

Comment:
Thanks Hans, but are those curly braces really useful? Why not just `return PositiveEnum::INVALID`? I'm not sure 🤷

Reply:
It might be implicit conversion without curly braces.

Reply:
@dzenanz  is correct.

Reply:
OK, that makes sense, thanks. But whenever implicit conversion is supported, the curly braces are unnecessary, right? For example, in a function that returns an `std::string`, it's still OK to `return "literal string"` without curly braces. Right?